test/templates/button/component-anchor.njk

```diff
@@ -5,6 +5,6 @@
 {%- macro anchor_button(properties={}, modifier={}, state={}, aria={}, classes="", size="") -%}
 {%- set is_default = modifier.block and modifier.block.length and not (modifier.block | join("")) %}
 
-<a class="{{ 'fd-button' if not modifier.block or modifier.block == "" or is_default }}{{ modifier.block | modifier('button') }}{{ ' fd-button--'+size if size }}{{ ' sap-icon--'+properties.icon if properties.icon }}{{ state | state }}{{ classes | classes }}"{{ aria | aria }}>{%- if properties.label %}{{properties.label}}{%- endif %}</a>
+<a class="{{ 'button' if not modifier.block or modifier.block == "" or is_default }}{{ modifier.block | modifier('button') }}{{ ' button--'+size if size }}{{ ' sap-icon--'+properties.icon if properties.icon }}{{ state | state }}{{ classes | classes }}"{{ aria | aria }}>{%- if properties.label %}{{properties.label}}{%- endif %}</a>
```

Comment:
What's this change for?  Seems unrelated to this PR.

Reply:
you are right, will go back to the original code, thanks.